### ISSUE TITLE
NTR: Gastric biopsy: carcinoid tumor (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9150]
New term request Gastric biopsy: carcinoid tumor (elthat_100321015445): Stomach  UMLS: 0349529

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000898
- label: Gastric carcinoid tumor
- parents: [Carcinoid tumor (HP:0100570)](https://hpo.jax.org/app/browse/term/HP:0100570)
- pmid: PMID:21603501
- def: Originating from enterochromaffin-like cells, gastric carcinoid tumors are rare tumors that develop within the gastric mucosa. They can present as an isolated lesion or there can be multiple lesions. The tumor can invade locally into deeper structures of the gastrointestinal tract wall. Solitary gastric carcinoids have a greater chance for the development of malignancy and metastasis as compared to multiple gastric carcinoids due to hypergastrinemia.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9150
- synonyms:
